tests/integration/devfile/cmd_devfile_env_test.go

```diff
@@ -28,7 +28,7 @@ var _ = Describe("odo devfile env command tests", func() {
 
 	Context("When executing env view", func() {
 		It("Should view all default parameters", func() {
-			helper.CmdShouldPass("odo", "create", "nodejs", "--project", commonVar.Project)
+			helper.CmdShouldPass("odo", "create", "nodejs", "--project", commonVar.Project, "--app", "testing")
```

Comment:
this is strange. the description is "should view all default parameters" but here you are setting `--app` explicitly.
Shouldn't this test verify that you can see the default application (app) in `odo env view` even if you don't set `--app`?

Reply:
done, fixed